.eslintrc.js

```diff
@@ -1,11 +1,18 @@
-// eslint config for .js files
```

Comment:
We need to work out what linting overrides we're going to want for TypeScript files particularly as it related to formatting, such as semi-colons. The current `.d.ts` definition files use semi-colons but there is no enforced linting preference. Presumably we'll want to insist on one way or the other as we do for the JS file.

Reply:
Can we use standard on ts files too, for consistency? I found this https://github.com/standard/eslint-config-standard-with-typescript

Reply:
I took a quick look. I think it can be done if we disable the rules requiring parser services (for now at least). I'm going to split some of this out into separate PRs as I feel like there's too much going on here now.

Reply:
First one: https://github.com/bugsnag/bugsnag-js/pull/711

Reply:
Second one: https://github.com/bugsnag/bugsnag-js/pull/712